README.md

```diff
@@ -325,11 +334,13 @@ $ kubectl get ClusterServiceVersion memcachedoperator.v0.0.1 -o json | jq '.stat
 After applying this manifest, nothing has happened yet, because the cluster does not meet the requirements specified in our manifest. Create the CustomResourceDefinition and RBAC rules for the `Memcached` type managed by the operator:
 
 ```sh
-$ kubectl apply -f deploy/rbac.yaml
-$ kubectl apply -f deploy/crd.yaml
+$ kubectl create -f deploy/crds/cache_v1alpha1_memcached_crd.yaml
```

Comment:
Is this how we are advertising how OLM be used? By applying the CSV directly, OLM does watch the operator and allow for updates, but does not achieve dependency resolution.
We should consider updating this with steps to install an operator via an operator-registry pod and catalogsource pair, since that is how operators are installed today via marketplace and operatorhub.io.
This can be documented in a future PR if need be

Reply:
@SamiSousa Agreed. For this PR I'm just fixing up the existing steps but we definitely need to update the OLM section of this guide to be in line with how we expect users to install their CSVs with CatalogSources and operator-registry.
Unless you want to overhaul this section I'm going to do a follow up PR for that after testing out the workflow myself.

Reply:
I'd be happy to help out with that :slightly_smiling_face: